73-ballan-wrasse/technical_specification.md

```diff
@@ -0,0 +1,70 @@
+# Access Request Management Improvements (Ballan Wrasse)
+
+**Epic Type:** Implementation Epic
+
+Epic planning and implementation follow the
+[Epic Planning and Marathon SOP](https://docs.ghga-dev.de/main/sops/sop001_epic_planning.html).
+
+## Scope
+
+### Outline
+
+The goal of this epic is to improve the management of access requests by giving
```

Comment:
Maybe we should also link the forum discussion(s) relevant to the epic?

Reply:
I think it's simpler to add any such links and other relevant info to the corresponding tasks in YouTrack.